src/containers/pages/FocusInvestigation/map/active/helpers/utils.tsx

```diff
@@ -480,3 +480,65 @@ export const buildOnClickHandler = (currentPlanId: string) => {
   }
   return clickHandler;
 };
+
+/**
+ * Map component popup handler
+ * @param map - mapbox map instance
+ * @param event - map event object
+ */
+
+export const buildMouseMoveHandler = (map: Map, event: EventData) => {
```

Comment:
Let's move this either to GisidaLite utils or the AssignmentMapWrapper utils.  I feel that it doesn't have much to do with FocusInvestigation